app/uk/gov/hmrc/emcstfe/connectors/httpParsers/ChrisXMLHttpParser.scala

```diff
@@ -30,7 +30,7 @@ import scala.xml.XML
 @Singleton
 class ChrisXMLHttpParser @Inject()(soapUtils: SoapUtils) extends Logging {
 
-  def rawXMLHttpReads[A](implicit xmlReads: XmlReader[A]): HttpReads[Either[ErrorResponse, A]] = (_: String, _: String, response: HttpResponse) => {
+  def rawXMLHttpReads[A](shouldExtractFromSoap: Boolean)(implicit xmlReads: XmlReader[A]): HttpReads[Either[ErrorResponse, A]] = (_: String, _: String, response: HttpResponse) => {
```

Comment:
This change is introduced as the response from submitting to ChRIS isn't in the same format as the response from ChRIS retrievals. There's probably a neater way to do this than trying to make one function generic but this works for now (open to discussion though).